editor.planx.uk/src/pages/FlowEditor/components/Settings/DesignSettings/FaviconForm.tsx

```diff
@@ -62,6 +62,7 @@ export const FaviconForm: React.FC<FormProps> = ({
           <InputRowItem width={formik.values.favicon ? 90 : 50}>
             <ImgInput
               img={formik.values.favicon || undefined}
+              backgroundColor={formik.values.primaryColour}
```

Comment:
_nit:_

I'm being super pedantic here but am I right in saying the favicon won't actually be set in a place where the background will be in effect? As in it'll be on a background which they've set their browser to?

Reply:
I'm just wondering whether the `primaryColour` might still not have enough contrast like we saw in the video with the OSL favicon?